### ISSUE TITLE
No Error Handling for empty playlists, missing fallbacks and alt text is generic

### ISSUE THREAD

**Issue by angelabera** [vaibhavrajharsh/Music-Player#21]
There are few issues here:-

- alt text is generic, i.e, it is empty
- unnecessary  nested<div> classes. Extra closing </div> tags near the bottom of .Home
- Fallback text for SVG icons or accessibility attributes like aria-hidden, title, etc.
- playlist-name and song-name may be redundant unless they have distinct purposes. `<h5 class="playlist-name song-name">Party Songs</h5>
`
- The <ul> under playlists is empty

@vaibhavrajharsh please assign me this issue, they need to resolved for proper structuring of the program
